letsencrypt/client/tests/reverter_test.py

```diff
@@ -0,0 +1,451 @@
+"""Test letsencrypt.client.reverter."""
+import logging
+import os
+import shutil
+import tempfile
+import unittest
+
+import mock
+
+from letsencrypt.client import errors
+
+
+class ReverterCheckpointLocalTest(unittest.TestCase):
+    # pylint: disable=too-many-instance-attributes
+    """Test the Reverter Class."""
+    def setUp(self):
+        from letsencrypt.client.reverter import Reverter
+
+        # Disable spurious errors... we are trying to test for them
+        logging.disable(logging.CRITICAL)
+
+        self.work_dir, self.direc = setup_work_direc()
+
+        self.reverter = Reverter(self.direc)
+
+        tup = setup_test_files()
+        self.config1, self.config2, self.dir1, self.dir2, self.sets = tup
+
+    def tearDown(self):
+        shutil.rmtree(self.work_dir)
+        shutil.rmtree(self.dir1)
+        shutil.rmtree(self.dir2)
+
+        logging.disable(logging.NOTSET)
```

Comment:
Same as in https://github.com/letsencrypt/lets-encrypt-preview/pull/185/files#r23518302